app/services/node-dashboard.js

```diff
@@ -2,13 +2,15 @@ const XlsxPopulate = require('xlsx-populate')
 const config = require('../../config')
 const log = require('./log')
 const dateFormatter = require('./date-formatter')
+const uploadDashboard = require('./dashboard/upload-dashboard')
 
 module.exports = function (reductions, capacity, formattedCaseloadData) {
-  const datestamp = dateFormatter.now().format('YYYYMMDDHHmmss')
+  const datestamp = dateFormatter.now().format('YYYYMMDDHH:mm:ss')
```

Comment:
Is this going to be a problem on wmt-web or do we have complete freedom with the filename?